extensions/nextjs-authkit/.env.example.append

```
@@ -0,0 +1,5 @@
+# WorkOS AuthKit
+WORKOS_API_KEY=sk_your_api_key
+WORKOS_CLIENT_ID=client_your_client_id
+WORKOS_COOKIE_PASSWORD=<generate 32+ char secret>
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)